sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -244,6 +244,7 @@ export const Constants: {
     readonly publisher: "x-opt-publisher-name";
     readonly viaPartitionKey: "x-opt-via-partition-key";
     readonly deadLetterSource: "x-opt-deadletter-source";
+    readonly messageState: "x-opt-message-state";
```

Comment:
Shouldnt we update the dependency on core-amqp in service-bus from `^3.0.0` to `^3.1.0` as Service Bus would need to read this property?